src/main/scala/edu/ie3/simona/model/participant/evcs/EvcsModel.scala

```diff
@@ -325,10 +325,10 @@ final case class EvcsModel(
         val time = tick.toDateTime(simulationStartDate)
 
         // separate into those entries that are still active
-        // and those that have ended before or at tick
+        // and those that have started before tick and will end after tick
         val (stillActive, endedEntries) = lastActiveEntries.partition {
           case (_, entry) =>
-            entry.tickStop > tick
+            (entry.tickStart < tick && entry.tickStop > tick)
```

Comment:
This also seems unnecessary, because entries that start after current tick cannot be present in `lastActiveEntries` anyway.

Reply:
Good catch. We missed that one when reverting that change above